### PR TITLE
(maint) Add platform definitions for vivid

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: configs/platforms/ubuntu-15.04-i386.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+platform "ubuntu-15.03-i386" do |plat|
```
Comment (haus) [inline configs/platforms/ubuntu-15.04-i386.rb]: 15.03 => 15.04

**Comment by haus** [comment]
:+1:

**Comment by melissa** [comment]
@haus once this is merged, should I tag puppetlabs-release as 0.9.3 and ship the new release packages?

**Comment by haus** [comment]
@melissa tagging and shipping seems sane. i don't think you need to ship anything except the new vivid packages though.

**Comment by haus** [comment]
or if you ship all of them, i don't think the other symlinks need to be repointed immediately.

**Comment by melissa** [comment]
Yeah, I wouldn't want to ship anything other than vivid for this

**Pull request merged**